### ISSUE TITLE
[jacobsen on 2008-05-01 01:16:41] : dom-mb: Implement icetop minbias hits

### ISSUE THREAD

**Issue by arcor** [WIPACrepo/domapp-tools-python#70]
Reopening for SVN rev 3199

**Comment by arcor** [comment]
[jacobsen on 2008-05-07 13:19:58]
From: 	  TStezelberger@lbl.gov
	Subject: 	minimum bias fpga
	Date: 	May 6, 2008 10:25:04 AM CDT
	To: 	  kael.hanson@gmail.com, john@mail.npxdesigns.com, tilav@udel.edu, seckel@bartol.udel.edu, tilo@bartol.udel.edu

Hi Everybody,

the minimum bias functionality is implemented in the domapp FPGA. The FPGA is checked in as build506 in the main branch and a build506-4xx on the rel-4xx branch.
Both the DOMAPP FPGA API document and the compressed data format document in docushare are updated.
The feature is disabled by default.

Finally as summary what is does.
1. on average every 1024th hit gets read out with a waveform (for normal runs this is the high gain channel only)
2. if the selected hit has a waveform readout already (beacon hit or lc) the minimum bias hit is deferred to the next hit.
3. The FPGA enforces ATWD A/B alternation.

Thorsten



-- 
------------------------------
Thorsten Stezelberger
Lawrence Berkeley National Lab
1 Cyclotron Road
Berkeley, CA 94720

(510) 495 2489

**Comment by arcor** [comment]
[jacobsen on 2008-06-16 17:59:35]
From: 	  gaisser@bartol.udel.edu
	Subject: 	implementation of Muon Calibration Filter
	Date: 	June 12, 2008 2:19:24 PM CDT
	To: 	  john@mail.npxdesigns.com, azriel@icecube.wisc.edu, krasberg@icecube.wisc.edu
	Cc: 	  tilo@bartol.udel.edu, tilav@bartol.udel.edu, seckel@bartol.udel.edu, kael.hanson@gmail.com, azriel@icecube.wisc.edu

All,
I had a conversation with Mark, John J and Azriel this morning
in which I emphasized the importance of implementing the
muon calibration filter as soon as possible.  They suggested
that it would be good to proceed with the 2 or 3 additional
tasks in parallel rather than in pipeline fashion.
John agreed to write the needed lines of code for COM
configuration.  Kael is away for a few days, but perhaps
he can ask someone else in his group to write the trigger.
As a reminder, I attach Tilo's proposal.
Cheers,
Tom
&#65532;

**Comment by arcor** [comment]
[jacobsen on 2008-06-18 01:54:04]
Sending        domapptest.py
Sending        domapptools/DeltaHit.py
Sending        domapptools/domapp.py
Transmitting file data ...
[commit 0edfcdf8f464acca97bbad19e884bbd6c56ba026] Committed revision 3198.

**Comment by arcor** [comment]
[jacobsen on 2008-06-18 01:56:21]
Checking in private/epxa10/hal/DOM_FPGA_domapp_regs.h;
/net/local/cvsroot/hal/private/epxa10/hal/DOM_FPGA_domapp_regs.h,v  <--  DOM_FPGA_domapp_regs.h
new revision: 1.1.1.6; previous revision: 1.1.1.5
done
Checking in private/epxa10/hal/domapp-hal.c;
/net/local/cvsroot/hal/private/epxa10/hal/domapp-hal.c,v  <--  domapp-hal.c
new revision: 1.1.1.14; previous revision: 1.1.1.13
done
/net/local/cvsroot/hal/.mailinglist,v  does not exist
Checking in public/hal/DOM_MB_domapp.h;
/net/local/cvsroot/hal/public/hal/DOM_MB_domapp.h,v  <--  DOM_MB_domapp.h
new revision: 1.1.1.20; previous revision: 1.1.1.19
done

**Comment by arcor** [comment]
[jacobsen on 2008-06-18 01:56:37]
Checking in ChangeLog;
/net/local/cvsroot/dom-ws/ChangeLog,v  <--  ChangeLog
new revision: 1.1.1.79; previous revision: 1.1.1.78
done
/net/local/cvsroot/dom-ws/.mailinglist,v  does not exist

**Comment by arcor** [comment]
[jacobsen on 2008-06-18 01:57:13]
cvs commit: Examining .
cvs commit: Examining private
cvs commit: Examining private/domapp
cvs commit: Examining resources
cvs commit: Examining resources/doc
Checking in private/domapp/DSCmessageAPIstatus.h;
/net/local/cvsroot/domapp/private/domapp/DSCmessageAPIstatus.h,v  <--  DSCmessageAPIstatus.h
new revision: 1.6.4.3; previous revision: 1.6.4.2
done
Checking in private/domapp/domSControl.c;
/net/local/cvsroot/domapp/private/domapp/domSControl.c,v  <--  domSControl.c
new revision: 1.21.4.17; previous revision: 1.21.4.16
done
/net/local/cvsroot/domapp/.mailinglist,v  does not exist
Checking in resources/doc/api.html;
/net/local/cvsroot/domapp/resources/doc/api.html,v  <--  api.html
new revision: 1.18.4.13; previous revision: 1.18.4.12
done

**Comment by arcor** [comment]
[jacobsen on 2008-06-18 01:59:30]
Sending        RELEASE_NOTES.txt
Transmitting file data .
[commit 0edfcdf8f464acca97bbad19e884bbd6c56ba026] Committed revision 3199.

**Comment by arcor** [comment]
[jkelley on 2020-06-01 13:33:59]
Batch close of resolved, released issues
